### PR TITLE
Added navbar and cleaned the code

### PR DESCRIPTION
**Pull request by eugenGirlescu**

### CODE REVIEW

File: src/main/java/com/car/carDealer/controller/Login.java [hunk 1]
```diff
@@ -30,33 +30,30 @@ public ModelAndView showLoginForm() {
...
-        user.setEmail(email);
-        user.setPassword(password);
+
         User valid = userService.findByEmailAndPassword(user.getEmail(), user.getPassword());
 
         if (valid != null) {
             mod.setViewName("hello");
         } else {
-            String msg = "Userul nu este inregistrat";
+            String msg = "Verificati emailul sau parola !";
```
Comment (andreisamoila74) [inline src/main/java/com/car/carDealer/controller/Login.java:49]: let's use english for everything

File: src/main/java/com/car/carDealer/controller/Login.java [hunk 2]
```diff
@@ -30,33 +30,30 @@ public ModelAndView showLoginForm() {
...
             model.addAttribute("msg", msg);
             mod.setViewName("user/login");
         }
         return mod;
     }
```
Comment (andreisamoila74) [inline src/main/java/com/car/carDealer/controller/Login.java:56]: remove all the empty lines

File: src/main/java/com/car/carDealer/model/Cars.java [hunk 3]
```diff
@@ -25,9 +25,12 @@
     @NotEmpty
     private String color;
 
-    @NotEmpty
     private String gearBox;
 
+    private int price;
+
+    private String coin;
```
Comment (andreisamoila74) [inline src/main/java/com/car/carDealer/model/Cars.java:32]: ?? what is coin?

File: src/main/java/com/car/carDealer/model/Cars.java [hunk 4]
```diff
@@ -83,4 +86,16 @@ public String getGearBox() {
     public void setGearBox(String gearBox) {
         this.gearBox = gearBox;
     }
+
+    public void setPrice(int price){this.price = price;}
```
Comment (andreisamoila74) [inline src/main/java/com/car/carDealer/model/Cars.java:90]: :| indentation

File: src/main/resources/templates/cars/add-car.html [hunk 5]
```diff
@@ -15,75 +15,147 @@
...
+                                <label for="price" class="col-form-label">Incarca poze :</label></legend>
+                            <form action="/action_page.php">
+                                <label for="img">Selecteaza imaginea :</label>
+                                <input type="file" id="img" name="img" accept="image/*" multiple>
+                            </form>
                         </div>
 
                         <div class="col-md-6">
-                            <input type="submit" class="btn btn-primary" value="Add Car">
+                            <input type="submit" class="btn btn-primary" value="Adauga masina">
```
Comment (andreisamoila74) [inline src/main/resources/templates/cars/add-car.html:145]: here too, let's use english

File: src/main/resources/templates/user/register.html [hunk 6]
```diff
@@ -1,20 +1,106 @@
...
+    height: 100%;
+    left: 0;
+    position: absolute;
+    top: 0;
+    width: 100%;
+    background-color: rgba(0,0,0,0.3);
+    z-index: -1;
+    border-radius: 10px;
+}
+
```
Comment (andreisamoila74) [inline src/main/resources/templates/user/register.html:34]: remove this empty lines

File: src/main/java/com/car/carDealer/model/Cars.java (hunk 3 above)
Comment (eugenGirlescu) [reply src/main/java/com/car/carDealer/model/Cars.java:32]: coin is "moneda"..EUR, USD, LEI

**Pull request merged**